test/unit/lib/plugins/aws/deployFunction.test.js

```diff
@@ -417,7 +417,47 @@ describe('AwsDeployFunction', () => {
           },
           Timeout: 12,
           MemorySize: 512,
-          Role: 'arn:aws:us-east-1:123456789012:role/role',
+          Role: 'my-role',
+        })
+      ).to.be.equal(true);
+    });
+
+    it('should inherit provider-level iam config', async () => {
```

Comment:
All new tests should be run with the use of `runServerless` utility as outlined here: https://github.com/serverless/serverless/blob/master/CONTRIBUTING.md#configuring-deprecations